invisible_cities/reco/calib_functions.py

```diff
@@ -130,3 +139,180 @@ def copy_sensor_table(h5in, h5out):
         if 'DataSiPM' in dIn.root.Sensors:
             datasipm = dIn.root.Sensors.DataSiPM
             datasipm.copy(newparent=group)
+
+
+def dark_scaler(dark_spectrum):
+    """
+    A function to scale dark spectrum with mu value.
+    """
+    def scaled_spectrum(x, mu):
+        return np.exp(-mu) * dark_spectrum
```

Comment:
I know, it is like that because we want to obtain poisson mu from it

Reply:
As far as I understand the x is required for a function to work with scipy.curve_fit (I'd have to study a bit more to see if I'm just being paranoid) and we preferred to use the fit_functions.fit method in all cases. We could simplify using scipy.leastsq but it would mean using a different wrapper for this seed fit. That being said, as far as I understand curve_fit uses leastsq so it wouldn't be too much of a problem

Reply:
Seems weird, but ok